src/main/java/org/terasology/blockNetwork/EfficientBlockNetwork.java

```diff
@@ -26,6 +26,10 @@
 import java.util.Iterator;
 import java.util.Set;
 
+/**
+ * Manages networks to represent blocks that connect in a networked manner.
```

Comment:
This could be phrased better. Something like "A class to represent and manage a network of connected blocks."